libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.ts

```diff
@@ -40,34 +59,48 @@ import { WattToastService } from '@energinet-datahub/watt/toast';
   imports: [
     WATT_TABS,
     WATT_CARD,
+    WATT_TABLE,
     TranslocoModule,
     WattButtonComponent,
     WattDatepickerComponent,
     ReactiveFormsModule,
     WATT_FORM_FIELD,
     WattDropdownComponent,
     DhPermissionRequiredDirective,
+    WattEmptyStateComponent,
+    CommonModule,
+    LetModule,
```

Comment:
nitpick: doesn't look like `LetModule` is used.

Reply:
You are right, it has been removed